code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -796,9 +796,6 @@ var/global/list/robot_modules = list(
 	robot.internals = new/obj/item/weapon/tank/jetpack/carbondioxide(src)
 	src.modules += robot.internals
 
-	src.emag = new /obj/item/weapon/gun/energy/plasmacutter/mounted(src)

```

Comment:
I would replace this with energy sword. They are really slow, so it would not be as easy to use energy sword. But at least give some means of self-defense

Reply:
An energy sword would be too much, I feel. Considering all the abilities they have, bolting doors, moving through airlocks without waiting. If someone manages to catch up to you, you deserve to be destroyed. They have a welder that does good enough damage, I think.

Reply:
Welder has limited fuel. And catching up to drones in straight hallway is really easy. I feel like they need something better than welder, but not as good as e-sword.

Reply:
I dunno, if some catches you in melee range, I think you deserve it. Do remember that this is a ghost role, still.

Reply:
But it shouldn't be dismissed. Emags have limited use. People choose to emag drones because they are less noticeable than borgs. But they shouldn't be defenseless.

Compared to borg, the only advantage drone has is the ability to hide and smaller icon. Otherwise borgs have more tools and have weapons, they also got more armour. 

It is easy to kill maint drone, which is fine. But when we emag it, it turnes from "nobody cares about ghost role" to an antag role which is meant to spice the round.